tutorials/3d/fps_tutorial/part_one.rst

```diff
@@ -71,28 +94,18 @@ First, go open the project settings and go to the "Input Map" tab. You'll find s
 actions have already been defined. We will be using these actions for our player.
 Feel free to change the keys bound to these actions if you want.
 
-While we still have the project settings open, quickly go check if MSAA (MultiSample Anti-Aliasing)
-is turned off. We want to make sure MSAA is off because otherwise we will get strange red lines
-between the tiles in our level later.
-
-.. tip:: The reason we get those red lines is because we are using lowpoly models
-        with low resolution textures. MSAA tries to reduce jagged edges between models and
-        because we are using lowpoly and low resolution textures in this project,
-        we need to turn it off to avoid texture bleeding.
-
-        A bonus with turning off MSAA is we get a more 'retro' looking result.
-
 _________
 
 Lets take a second to see what we have in the starter assets.
 
-Included in the starter assets are five scenes: ``BulletScene.tscn``, ``Player.tscn``,
-``SimpleAudioPlayer.tscn``, ``TestingArea.tscn``, and ``TestLevel.tscn``.
+Included in the starter assets are several scenes. For example, in `res://` we have 14 scenes, most of which we will be visiting as
+we go through this tutorial series.
 
-We will visit all of these scenes later, but for now open up ``Player.tscn``.
+For now lets open up ``Player.tscn``.
```

Comment:
should be `let's`

Reply:
There's too much to review. I won't be able to look at all of it at once. I'll revisit this some time and I'll let you know if I notice. Thank you very much for contributing.

Reply:
Thanks for reviewing! I'll fix these soon :)